src/apple/system.rs

```diff
@@ -20,10 +28,7 @@ use std::sync::Arc;
 
 use libc::{self, c_int, c_void, size_t, sysconf, _SC_PAGESIZE};
 
-use core_foundation_sys::base::{kCFAllocatorDefault, CFRelease};
-
 use utils::into_iter;
-
 // We need to wrap `DASessionRef` to be sure `System` remains Send+Sync.
 struct SessionWrap(ffi::DASessionRef);
```

Comment:
I think you need to `cfg` this type too.

Reply:
I thought so as well but `cargo check --target=aarch64-apple-ios` didn't seem to care:
```
cargo check --target=aarch64-apple-ios
   Compiling sysinfo v0.15.8 (/Users/user/dev/github/sysinfo)
warning: dropping unsupported crate type `cdylib` for target `aarch64-apple-ios`

warning: 1 warning emitted

    Finished dev [unoptimized + debuginfo] target(s) in 1.06s
 ```

Reply:
Dark magic.

Reply:
I moved `SessionWrap` to `macos::ffi` so it doesn't exist on iOS anymore.

Reply:
Thanks!